src/main/kotlin/com/ampnet/userservice/grpc/GrpcUserServer.kt

```diff
@@ -111,6 +112,20 @@ class GrpcUserServer(
         )
     }
 
+    override fun getUsersByEmail(request: GetUsersByEmailRequest, responseObserver: StreamObserver<UsersResponse>) {
+        logger.debug { "Received gRPC request getUsersByEmail: $request" }
```

Comment:
Check the function toString for this class. Not sure how it will look in the logs.